docs/docs/basic-usage.md

```diff
@@ -64,6 +64,24 @@ $ poetry add pendulum
 It will automatically find a suitable version constraint **and install** the package and subdependencies.
 
 
+## Using your virtual environment
+
+By default, poetry creates a virtual environment in `{cache-dir}/virtualenvs` (`{cache-dir}\virtualenvs` on Windows).
```

Comment:
Might be a good idea to reference the default value of `cache-dir` here. And also add a note about the ability to configure `venv.in-project` and how that changes this.

Reply:
I linked to their headers on the config doc's page. The links aren't working from GitHub but they have the same format as the other links that are working on the poetry website.